docs/API.md

```diff
@@ -1702,34 +1703,33 @@ The above example is equivalent to the `meta/common-info` example above.
 
 **Not yet implemented**
 
-#### POST *id*/resources/name.stream
+#### GET *id*/meta/resources
 
-Posting to the resources path creates a new version of the given stream
-for the charm with the given id. The request returns the new version.
+Getting from this path will return metadata regarding the resources for the charm with id of `id`.
```

Comment:
To fit with the standard phrasing used in other places:

    The `meta/resources` path returns information on all the resources associated with the given charm *id*.